scripts/zephyr_module.py

```diff
@@ -19,7 +19,7 @@
 import pykwalify.core
 import subprocess
 import re
-
+from pathlib import PurePath
```

Comment:
Another option is to `import posixpath` and then do `posixpath.abspath(kconfig_file)`, etc., if you consistently want to deal with *nix paths.

`os.path` is `posixpath` if you're on *nix, and `ntpath` if you're on Windows.

Reply:
yeah, that could also be a choice.
Only reason for deciding on `PurePath` here, is because it is what is also used in `west`.